Fabric.Authorization.API/scripts/run-client-functional-tests.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
+IMAGE="0"
+
+# call script with -i to use the docker hub image
+for i in "$@"
+do
+case $i in
+    -i)
+    IMAGE="1";;
+    *)
+          # unknown option
```

Comment:
Does it make sense to print out an error message here if someone attempts to use an unknown argument/option?

Reply:
something like invalid args automatically print out the help arg